access_utils/general_stats_parse.cwl

```diff
@@ -6,10 +6,10 @@ $namespaces:
   foaf: 'http://xmlns.com/foaf/0.1/'
 id: general_stats_parse
 label: general_stats_parse
-baseCommand: general_stats_parse.py
+baseCommand: general_stats_parse.py .
```

Comment:
what is the `.` for? i think that might cause it to fail

Reply:
ah good catch! thanks

Reply:
so the reason I have that is because the output of aggregate is a list of directories, and this tool just takes a single dir

so I'm putting that list of directories in the CWD of the tool, and using `--dir .` as the tools argument